tools/fregrid/fregrid.c

```diff
@@ -106,15 +105,15 @@ char *usage[] = {
   "                                                                                      ",
   "--scalar_field    scalar_fld  specify the scalar field name to be regridded. The      ",
   "                              multiple entry field names are seperated by comma.      ",
-  "                                                                                      ",    
+  "                                                                                      ",
   "--u_field         u_fld       specify the vector field u-componentname to be          ",
   "                              regridded. The multiple entry field names are seperated ",
   "                              by comma. u_field must be paired together with v_field. ",
```

Comment:
This maybe makes it a little clearer that these should be components of spherical vector fields:
```suggestion
  "                              by comma. u_field must be paired together with v_field. ",
  "                              u_field corresponds to the zonal component of the         ",
  "                              vector field.                                                                           ",
```

Reply:
Thanks. Will usw suggestion